makefile

```diff
@@ -1,4 +1,4 @@
-version = 0_0_00
+version = ${shell php -r "echo str_replace('.', '_', json_decode(file_get_contents('Newsletter2Go/Export/composer.json'))->version);"}
```

Comment:
**Reason:** Now make will create a file with the correct version by default (old system of `make version=XXXX` still works though).